src/main/java/com/linkedin/kmf/services/MultiClusterTopicManagementService.java

```diff
@@ -66,21 +66,23 @@
  * - Increase partition number of the monitor topic if either partitionsToBrokersRatio or minPartitionNum is not satisfied
  * - Increase replication factor of the monitor topic if the user-specified replicationFactor is not satisfied
  * - Reassign partition across brokers to make sure each broker acts as preferred leader of at least one partition of the monitor topic
- * - Trigger preferred leader election to make sure each broker acts as leader of at least one partition of the monitor topic.
+ * - Trigger preferred leader election to make sure each broker acts as the leader of at least one partition of the monitor topic.
  * - Make sure the number of partitions of the monitor topic is same across all monitored clusters.
  *
  */
 public class MultiClusterTopicManagementService implements Service {
   private static final Logger LOG = LoggerFactory.getLogger(MultiClusterTopicManagementService.class);
   private static final String METRIC_GROUP_NAME = "topic-management-service";
+  private final CompletableFuture<Void> _topicPartitionReady = new CompletableFuture<>();
   private final AtomicBoolean _isRunning = new AtomicBoolean(false);
   private final String _serviceName;
   private final Map<String, TopicManagementHelper> _topicManagementByCluster;
   private final int _scheduleIntervalMs;
   private final long _preferredLeaderElectionIntervalMs;
   private final ScheduledExecutorService _executor;
-  final private CompletableFuture<Void> _completableFuture;
+  private final CompletableFuture<Void> _completableFuture;
```

Comment:
Again, can we not name a variable by its type (unless there is a very good reason for doing it)?